src/raylib/raylib.mli

```diff
@@ -3011,3 +3011,5 @@ val set_audio_stream_pitch : AudioStream.t -> float -> unit
 
 val set_audio_stream_buffer_size_default : int -> unit
 (** [set_audio_stream_buffer_size_default size] Default size for new audio streams*)
+
+module Rl = Rlgl
```

Comment:
I don't think the abbreviation makes much sense here. `Rlgl` stands for "Raylib GL", no?
I think `Rlgl` is fine here, since it's named like that throughout the raylib C library. Otherwise, I'd prefer `Gl`, so we can access it like `Raylib.Gl`, which fits `rlgl`.

Reply:
Sure, I chose that because of the rl_ prefix but that doesn't really make that much sense in retrospect.